agent/consul/peering_backend_test.go

```diff
@@ -60,6 +65,83 @@ func TestPeeringBackend_ForwardToLeader(t *testing.T) {
 	})
 }
 
+func TestPeeringBackend_GetServerAddresses(t *testing.T) {
+	if testing.Short() {
+		t.Skip("too slow for testing.Short")
+	}
+
+	_, cfg := testServerConfig(t)
+	cfg.GRPCTLSPort = 8505
```

Comment:
Will it be an issue if this port is in use by other tests? Could use `freeport.GetOne`